Subject: [PATCH] audioflinger: increase max tracks per uid

Fix some apps from not being able to initialize audio tracks
and prevent the following error, "AudioFlinger could not create
track. status: -12".

diff --git a/services/audioflinger/Threads.h b/services/audioflinger/Threads.h
index 0e60119..5539aad 100644
--- a/services/audioflinger/Threads.h
+++ b/services/audioflinger/Threads.h
@@ -510,8 +510,8 @@ class PlaybackThread : public ThreadBase {
     static const int8_t kMaxTrackRetriesOffload = 20;
     static const int8_t kMaxTrackStartupRetriesOffload = 100;
     static const int8_t kMaxTrackStopRetriesOffload = 2;
-    // 14 tracks max per client allows for 2 misbehaving application leaving 4 available tracks.
-    static const uint32_t kMaxTracksPerUid = 14;
+    // 18 tracks max per client allows for 2 misbehaving application leaving 8 available tracks.
+    static const uint32_t kMaxTracksPerUid = 40;
 
     PlaybackThread(const sp<AudioFlinger>& audioFlinger, AudioStreamOut* output,
                    audio_io_handle_t id, audio_devices_t device, type_t type, bool systemReady);
